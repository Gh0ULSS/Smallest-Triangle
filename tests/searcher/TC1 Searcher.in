34.54, 12.15