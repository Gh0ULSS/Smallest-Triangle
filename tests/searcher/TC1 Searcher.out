read 1 points
34.54, 12.15
This is not a triangle